desktopexporter/internal/server/server.go

```diff
@@ -51,8 +51,17 @@ func (s *Server) Handler() http.Handler {
 	router.HandleFunc("GET /api/traces", s.tracesHandler)
 	router.HandleFunc("GET /api/traces/{id}", s.traceIDHandler)
 	router.HandleFunc("GET /api/sampleData", s.sampleDataHandler)
-	router.HandleFunc("GET /api/clearData", s.clearTracesHandler)
+	router.HandleFunc("GET /api/clearTraces", s.clearTracesHandler)
 	router.HandleFunc("GET /traces/{id}", s.indexHandler)
+	
+	// Feature flag for logs frontend route
+	if os.Getenv("ENABLE_LOGS") == "true" {
+		router.HandleFunc("GET /logs", s.indexHandler)
+		router.HandleFunc("GET /api/logs", s.logsHandler)
+		router.HandleFunc("GET /api/logs/{id}", s.logIDHandler)
+		router.HandleFunc("GET /api/logs/trace/{id}", s.logsByTraceHandler)
+		router.HandleFunc("GET /api/clearLogs", s.clearLogsHandler)
```

Comment:
I'm unsure about this API structure, but let's roll with it for now. I expect this to change as we try to build a UI